docs/source/isaura.rst

```diff
@@ -1,5 +1,168 @@
+
+
+To compute the position of the blobs, we need to find the two extreme voxels of the track, which is done following the BFS algorithm. Then, the energy-weighted averaged position of the hits inside these voxels will correspond to the **blob center**, as illustrated in the figure below.
+
+
+.. image:: images/isaura/blobs_position_definition.png
+   :width: 400
+   :align: center
+
+
+In consequence, the first thing to do is to localize the two end voxels for each track. Defining the distance between any pair of voxels as the shortest path **along the track** that connects them, the two extreme voxels will be the ones with the longest distance between them. However, there are two special cases that are important to comment:
+
+
+ - It is possible that some spurious **low-energy** hits appear around the track (due to over-iterations during the Richardson-Lucy deconvolution process, as commented in :doc:`beersheba`; or some noise inside the chamber, for example). If these hits are reconstructed around the track but not far enough to produce a different S2 or track (taking into account the voxel size), they can be considered as a part of the main one and, being a bit separate, it is probable that they end up belonging to an extreme voxel. That case would not be correct, and in order to solve it, the voxel will be dropped from the track and its energy passed to the closest one. This process is only carried out if the voxel energy is lower than ``energy_threshold`` and the track is made by more than ``min_voxels`` voxels. Once this procedure is done, the extreme voxels are searched and found again recursively, until none of these conditions are fulfilled.
+
+ - Another particular scenario is the one that comes up when there are multiple end-voxel candidates (one can imagine that the shorter the track the more probable is this to happen). To deal with it, the more energetic candidates will be the ones set as extremes. With this convention, we aim to minimize the use of the voxel-dropping algorithm commented above for those cases where the energy of one candidate is larger than ``energy_threshold`` while the other one is below that value. 
+
+
+Once the extreme voxels are properly found, the center position of the blobs --stored in the ``Tracks/Tracking`` table as: ``blobi_x``, ``blobi_y``, and ``blobi_z``, (with ``i`` being 1, 2), respectively-- is computed in accordance with the figure previously presented.
```

Comment:
I would add here the reference to the footnote about the definition of Blob 1 and Blob 2 (you do it in the following paragraph)

Reply:
When the position of the blobs is computed, labels 1 and 2 are selected randomly. They are reassigned once energies are calculated. Therefore, I would not add the reference here, since ``eblob1`` and ``eblob2`` parameters are not defined yet.